Services/CO.CDP.EntityVerification/Persistence/DatabasePponRepository.cs

```diff
@@ -51,6 +51,11 @@ public void Dispose()
                 (scheme == IdentifierSchemes.Ppon && p.IdentifierId == id));
     }
 
+    public async Task<IEnumerable<IdentifierRegistries>> GetIdentifierRegistriesAsync(string countryCode)
+    {
+        return await context.IdentifierRegistries.Where(q => q.CountryCode.ToUpper() == countryCode.ToUpper()).ToListAsync();
+    }
```

Comment:
Missing test case.

Reply:
Also, since we query by the country code we will need a non-unique index on this field in the database.